battery_management_system/template.py

```diff
@@ -0,0 +1,32 @@
+from battery_management_system.battery_states import BatteryState
+from configparser import ConfigParser
+
+class BaseTemplate:
+    def __init__(self, value) -> None:
+        # configparser is kinda limited in auto serialization but servers fine for this simple example
+        self.config = ConfigParser()
```

Comment:
Separate .ini file contents and let each parameter sub class read respective .ini file

BaseTemplate interface is Fat, remove min and max